ferris_elf/lib.py

```diff
@@ -81,17 +81,21 @@ async def benchmark(
 
         verified_results = [r for r in results if r.verified]
         if len(verified_results) > 0:
-            median = stats.mean([r.median for r in verified_results])
-            average = stats.mean([r.average for r in verified_results])
+            median = Picoseconds.from_picos(
+                stats.mean([r.median.as_picos() for r in verified_results])
+            )
+            average = Picoseconds.from_picos(
+                stats.mean([r.average.as_picos() for r in verified_results])
+            )
             await ctx.reply(
                 embed=discord.Embed(
                     title="Benchmark complete (Verified)",
                     description=f"Median: **{median}**\nAverage: **{average}**",
                 )
             )
         else:
-            median = stats.mean([r.median for r in results])
-            average = stats.mean([r.average for r in results])
+            median = Picoseconds.from_picos(stats.mean([r.median.as_picos() for r in results]))
+            average = Picoseconds.from_picos(stats.mean([r.average.as_picos() for r in results]))
```

Comment:
Im a little worried about mypy here, statistics very obviously returns a float here on int args, but mypy is happy to call it an int, and then in from_picos we do no proper casting to `int()` cause we assume the argument is well formed
mypy is :trollface: yet again

Reply:
Committing your suggested change should take care of this.